tests/models/bloom/test_modeling_bloom.py

```diff
@@ -363,177 +188,43 @@ def test_bloom_lm_head_model(self):
-            tokenizer.decode(greedy_output[1], skip_special_tokens=True),
-        )
-
-    @slow
-    @require_torch_accelerator
-    def test_batch_generation_padding(self):
-        path_560m = "bigscience/bloom-560m"
-        model = BloomForCausalLM.from_pretrained(path_560m, use_cache=True, revision="gs555750").to(torch_device)
-        model = model.eval()
-        tokenizer = BloomTokenizerFast.from_pretrained(path_560m, padding_side="left")
-
-        input_sentence = ["I enjoy walking with my cute dog", "Hello my name is"]
-        input_sentence_without_pad = "Hello my name is"
-
-        input_ids = tokenizer.batch_encode_plus(input_sentence, return_tensors="pt", padding=True)
-        input_ids_without_pad = tokenizer.encode(input_sentence_without_pad, return_tensors="pt")
-
-        input_ids, attention_mask = input_ids["input_ids"].to(torch_device), input_ids["attention_mask"]
-        greedy_output = model.generate(input_ids, attention_mask=attention_mask, max_length=50, do_sample=False)
-        greedy_output_without_pad = model.generate(
-            input_ids_without_pad.to(torch_device), max_length=50, do_sample=False
-        )
-
-        # test token values
-        self.assertEqual(greedy_output[-1, 3:].tolist(), greedy_output_without_pad[0, :-3].tolist())
-
-        # test reconstructions
-        self.assertEqual(
-            tokenizer.decode(greedy_output[-1, 3:], skip_special_tokens=True),
-            tokenizer.decode(greedy_output_without_pad[0, :-3], skip_special_tokens=True),
-        )
-
-    @slow
-    @require_torch_accelerator
-    def test_batch_generated_text(self):
-        path_560m = "bigscience/bloom-560m"
-
-        model = BloomForCausalLM.from_pretrained(path_560m, use_cache=True, revision="gs555750").to(torch_device)
-        model = model.eval()
-        tokenizer = BloomTokenizerFast.from_pretrained(path_560m, padding_side="left")
-
-        input_sentences = [
-            "Hello what is",
-            "Running a quick test with the",
-        ]
-        inputs = tokenizer(input_sentences, return_tensors="pt", padding=True, truncation=True)
-        generated_ids = model.generate(
-            inputs["input_ids"].to(torch_device), attention_mask=inputs["attention_mask"], max_length=20
-        )
-        generated_text = tokenizer.batch_decode(generated_ids, skip_special_tokens=True)
-
-        # these generations match those of the PyTorch model
-        EXPECTED_GENERATIONS = [
-            "Hello what is the best way to get the data from the server? I have tried",
-            "Running a quick test with the following command:\nsudo apt-get install python3\nsudo apt-get install python2",
-        ]
-
-        self.assertListEqual(generated_text, EXPECTED_GENERATIONS)
-
     @unittest.skip("Bloom needs a 2D attention for alibi")
     def test_custom_4d_attention_mask(self):
         pass
 
 
 @require_torch
-class BloomEmbeddingTest(unittest.TestCase):
-    """
```

Comment:
this comment pertains the first test only, not the whole tester.

also, the tester name was innacurate